src/components/FilterControls/MultiSelect/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useEffect } from 'react'
+import OutlinedInput from '@mui/material/OutlinedInput'
+import InputLabel from '@mui/material/InputLabel'
+import MenuItem from '@mui/material/MenuItem'
+import FormControl from '@mui/material/FormControl'
+import Select, { SelectChangeEvent } from '@mui/material/Select'
+import Checkbox from '@mui/material/Checkbox'
+import ListItemText from '@mui/material/ListItemText'
+
+interface MultiSelectProps {
+  options: string[]
+  defaultOptions: string[]
```

Comment:
Default options should be constant through the lifetime of your app, so you can either inline this below or remove it entirely, since you don't seem to need it.